docs/source/markdown/podman-generate-systemd.1.md

```diff
@@ -60,6 +60,10 @@ Note: On a system with SELinux enabled, the generated files will inherit context
 
 Print the created units in specified format (json). If `--files` is specified the paths to the created files will be printed instead of the unit content.
 
+#### **--header**=*header*
+
+Append the specified header.
```

Comment:
It may be worth elaborating that it can be used in combination with `--no-header`.